CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 * **Fix `partition_html()` swallowing some paragraphs**. The `partition_html()` only considers elements with limited depth to avoid becoming the text representation of a giant div. This fix increases the limit value.
 * **Fix SFTP** Adds flag options to SFTP connector on whether to use ssh keys / agent, with flag values defaulting to False. This is to prevent looking for ssh files when using username and password. Currently, username and password are required, making that always the case.
+* **Fix Add Mongodb Env Vars to ingest-test-fixtures-update-pr** Mongodb env vars were never added. This caused errors when trying to update test fixtures.
```

Comment:
non-code / CI bits don't need to bump the changelog

Reply:
@ryannikolaidis  I removed all changes to changelog and `__version__`. I hope thats what you meant.

Reply:
yep, thank you!